xbmc/cores/AudioEngine/Sinks/windows/AESinkFactoryWin32.cpp

```diff
@@ -130,14 +130,11 @@ std::vector<RendererDetail> CAESinkFactoryWin::GetRendererDetails()
     PropVariantClear(&varName);
 
     hr = pProperty->GetValue(PKEY_Device_EnumeratorName, &varName);
-    if (FAILED(hr))
+    if (SUCCEEDED(hr) && varName.pwszVal != nullptr)
     {
-      CLog::LogF(LOGERROR, "Retrieval of endpoint enumerator name failed.");
-      goto failed;
+      details.strDeviceEnumerator = KODI::PLATFORM::WINDOWS::FromW(varName.pwszVal);
+      StringUtils::ToUpper(details.strDeviceEnumerator);
```

Comment:
ok to make optional, but please keep logging errors of GetValue and null enumerator string.